### ISSUE TITLE
For reducing time dealing copied content

### ISSUE THREAD

**Issue by Sarcastic-Programmer** [seahindeniz/BrainlyTools_Extension#350]
**Is your feature request related to a problem? Please describe.**
 
NO

**Describe the solution/idea you'd like**

A button name "Check for plagiarism" can be added in the ticket itself which will compare the ticket **content** to the Google or other websites if found plagiarized then it will automatically copy the link to clipboard from where the content is plagiarized.

**Describe expected outcome from your solution**

It will save time for checking plagiarism of every content.

**Is it possible?**

**Comment by seahindeniz** [comment]
Hey @Sarcastic-Programmer,

I totally agree with you regarding the benefits of this feature in terms of time.
However, we already have considered this feature for a long time ago and sadly, we can't have it because there are several reasons where this cannot be applied. 

Let's say you usually check answers by looking at the content and recognize the part of where the actual content starts and ends. You just copy that part and search it on Google and/or other services. That service is mainly free for users.

We can't have it because:
- there might be some difficulties, unreliable results while using this feature with English and other languages as well.
- extension cannot understand the exact location of the actual content that needs to be reviewed/searched. (Considering that, users are usually posting greetings, endings that are not particularly related to their posts)
- plagiarism services do not accept requests from external services without charging extra fees- even if the extension user may provide the target content by simply selecting/highlighting the actual part.
- cooperation with external services or extensions is not possible either even with paid solutions. We have too many extension users who are willing to use this feature thus which results in too much activity that we may, can't afford 😓

So basically, it's not possible at the moment. If a big company such as Google may share its resources with open source projects in the future, we may, can benefit from it.

If you have any suggestion where my reasons get false, please do share. I'll like to take a look and research to make this feature possible.

**Comment by CoolestCat015** [comment]
@seahindeniz https://www.reddit.com/r/opensource/comments/8xgldr/any_freeopensource_software_for_plagiarism/

**Comment by seahindeniz** [comment]
Thanks @CoolestCat015 most of the suggestions in that link are for cross-comparing multiple documents. One of them is based on Google search which is also needs to be payed for more than 100 searches, daily.

https://developers.google.com/custom-search/v1/overview
